rest-backend/src/main/java/com/sap/psr/vulas/backend/repo/LibraryRepositoryImpl.java

```diff
@@ -88,15 +87,15 @@ public Library customSave(Library _lib) {
 			_lib.setCreatedAt(managed_lib.getCreatedAt());
 			_lib.setModifiedAt(Calendar.getInstance());
 			
-			//	Re-create wellknownDigest if it is null in our current db (this part should be removed once it is created for all)
-			if(managed_lib.getWellknownDigest()==null || (managed_lib.getDigestTimestamp()==null && managed_lib.getWellknownDigest())) {
-				_lib.verifyDigest();
-			}
-			else {
-				_lib.setWellknownDigest(managed_lib.getWellknownDigest());
-				_lib.setDigestVerificationUrl(managed_lib.getDigestVerificationUrl());
-				_lib.setDigestTimestamp(managed_lib.getDigestTimestamp());
-			}
+//			//	Re-create wellknownDigest if it is null in our current db (this part should be removed once it is created for all)
```

Comment:
Just delete all those commented sections :)

Reply:
you are right, done!